app/models/user.rb

```diff
@@ -1,8 +1,9 @@
 class User < ActiveRecord::Base
   def self.from_omniauth(auth)
-    where(provider: auth.provider, uid: auth.uid).first_or_create do |user|
+    where(provider: auth.provider, email: auth.info.email).first_or_create do |user|
```

Comment:
I believe we will want to stick with keying from UID; let's discuss after standup! 💁